### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by justinvandenberg**
Not much here to actually QA. Basically just setting up the Tailwind config, `wp-config.php`, and `README`. had to add one of our plugins instead of including it as a submodule to work on Pantheon which doesn't support them.

### CODE REVIEW

File: config/tailwindcss/tailwindcss.config.js [hunk 1]
```diff
@@ -270,20 +134,14 @@ module.exports = {
       auto: 'auto',
     },
     letterSpacing: {
-      tighter: '-0.05em',
-      tight: '-0.025em',
       normal: '0',
-      wide: '0.025em',
-      wider: '0.05em',
-      widest: '0.1em',
+      wide: '0.1em',
```
Comment (olets) [inline config/tailwindcss/tailwindcss.config.js]: thinking out loud: interesting hadn't noticed what units we default to. I've been considering using ch for letter spacing

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "VigetPress",
+  "name": "NPM",
   "version": "1.0.6",
   "author": "<justinjvandenberg@gmail.com>",
```
Comment (olets) [inline package.json]: not in this PR I know, but wonder if this should be a Viget email?

Comment (justinvandenberg) [reply package.json]: Whoops, good catch!

File: config/tailwindcss/tailwindcss.config.js (hunk 1 above)
Comment (justinvandenberg) [reply config/tailwindcss/tailwindcss.config.js]: Does `ch` work on IE? I was under the impression that it doesn't.

File: package.json (hunk 2 above)
Comment (justinvandenberg) [reply package.json]: Whoops, good catch.

File: config/tailwindcss/tailwindcss.config.js (hunk 1 above)
Comment (olets) [reply config/tailwindcss/tailwindcss.config.js]: Ah, dunno

**Pull request merged**